application/asset/sass/_screen.scss

```diff
@@ -3465,6 +3466,14 @@ body.sidebar-open #content {
         display: table;
     }
     
+    #user-selector.empty .users-available {
```

Comment:
Just from an organizational standpoint, presumably this should go in the User Permissions section just below this?